client/conn.go

```diff
@@ -67,14 +79,41 @@ func newConn(cfg *Config) (c *conn, err error) {
 		queries:     make([]types.Query, 0),
 	}
 
-	var peers *proto.Peers
 	// get peers from BP
+	var peers *proto.Peers
 	if peers, err = cacheGetPeers(c.dbID, c.privKey); err != nil {
 		log.WithError(err).Error("cacheGetPeers failed")
 		c = nil
 		return
 	}
-	c.pCaller = rpc.NewPersistentCaller(peers.Leader)
+
+	if cfg.UseLeader {
+		c.pCaller = rpc.NewPersistentCaller(peers.Leader)
+		c.leader = &pconn{
+			parent:  c,
+			pCaller: c.pCaller,
+		}
+	}
+
+	// choose a random follower node
+	if cfg.UseFollower && len(peers.Servers) > 1 {
+		for {
+			rand.Seed(time.Now().UnixNano())
```

Comment:
It's inappropriate for client SDK to modify the golang runtime global random source seed.
I thought using client package global `randSource` is better, already initialized with `rand.New(rand.NewSource(time.Now().UnixNano()))`.